specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json

```
@@ -1595,8 +1668,8 @@
         "parameters": [
           {
             "in": "query",
-            "name": "status",
-            "description": "(Optional) If specified, select workers by worker status.",
+            "name": "state",
+            "description": "If specified, select workers by worker status.",
```

Rename to `state` to be consistent with the query name?
```suggestion
            "description": "If specified, select workers by worker state.",
```